go/vt/servenv/grpc_server.go

```diff
@@ -86,7 +86,7 @@ var (
 
 	// EnforcementPolicy MinTime that sets the keepalive enforcement policy on the server.
 	// This is the minimum amount of time a client should wait before sending a keepalive ping.
-	GRPCKeepAliveEnforcementPolicyMinTime = flag.Duration("grpc_server_keepalive_enforcement_policy_min_time", 5*time.Minute, "gRPC server minimum keepalive time")
+	GRPCKeepAliveEnforcementPolicyMinTime = flag.Duration("grpc_server_keepalive_enforcement_policy_min_time", 10*time.Second, "gRPC server minimum keepalive time")
```

Comment:
I think in addition to changing the default, we should also force the value if it violates the constraint we want to impose (that PolicyMinTime <= grpc_keepalive_timeout). Line 157.

Reply:
Hmm I'm not sure if it should be up to us to do this kind of validations. I think that could be a slippery slope. Two things come to mind:
* I think we already have many other settings that could create conflict if not configured correctly that we are not doing this. 
* Even if we add that validation we can't prevent that vtgates are configured incorrectly in respect to the flags that the user  provides to the tablet (e.g you could configure the vtgates to be 10s and enforcement policy of 10s; then tablets to be 5s / 5s. The constraint you are proposing won't catch that.

Reply:
Valid points. We can merge this as-is.